web/packages/teleport/src/stores/storeNotifications.ts

```diff
@@ -33,25 +35,58 @@ export type Notification = {
   item: AccessListNotification;
   id: string;
   date: Date;
+  clicked?: boolean;
 };
 
 // TODO?: based on a feedback, consider representing
 // notifications as a collection of maps indexed by id
 // which is then converted to a sorted list as needed
 // (may be easier to work with)
-export type NotificationState = {
+export type NotificationStoreState = {
   notifications: Notification[];
 };
 
-const defaultNotificationState: NotificationState = {
+const defaultNotificationStoreState: NotificationStoreState = {
   notifications: [],
 };
 
-export class StoreNotifications extends Store<NotificationState> {
-  state: NotificationState = defaultNotificationState;
+type LocalNotificationStates = {
+  clicked: string[];
+  hidden: string[];
+};
+
+const defaultLocalNotificationStates: LocalNotificationStates = {
+  clicked: [],
+  hidden: [],
+};
 
-  getNotifications() {
-    return this.state.notifications;
+export class StoreNotifications extends Store<NotificationStoreState> {
+  state: NotificationStoreState = defaultNotificationStoreState;
+
+  getNotifications(): Notification[] {
```

Comment:
see my comment about subscribing to the store and when updates to store happens. i think we also need to add logic to `setNotifications` to update the local storage when store updates?